### PR TITLE
Enable ssl-enforcer in production

### PR DESCRIPTION
**Pull request by joshuatobin**
CC: @heroku/sre

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  require 'rack/ssl-enforcer'
```
Comment (reidmix) [inline config/environments/production.rb]: should be already loaded (required) in Gemfile.

**Comment by reidmix** [comment]
Aside from removing the required, rest looks good!

**Comment by lexelby** [comment]
Holding off on merging this until we have our cert in order for staging and production.

**Comment by jmervine** [comment]
👍

**Comment by reidmix** [comment]
👍

**Pull request merged**